### PR TITLE
Add energy&meteo systems jobs

### PR DESCRIPTION
**Pull request by Kislovskiy**

### CODE REVIEW

File: data/pages-content/jobs-board.md [hunk 1]
```diff
@@ -392,3 +392,135 @@ Kiwi.com uncovers cheap travel options that others simply can't find.
...
+  - Container Platform (Docker, Kubernetes)
+  - Microservices
+
+  **Your profile:**
+
+  You want to think and question, are willing to lend a hand and solve problems. These features make you stand out:
+
+  - You have completed a scientific degree in the field of computer science, software engineering, physics or mathematics or an apprenticeship as an IT specialist for application development (f/m/d) or have a comparable qualification - cross-entry is also possible.
+  - You are equally welcome as a career starter (f/m/d) or with professional experience.
+  -  You have experience in an object-oriented or multi-paradigm programming language such as Python or Java.
```
Comment (hypha) [inline data/pages-content/jobs-board.md:428]:
```suggestion
  - You have experience in an object-oriented or multi-paradigm programming language such as Python or Java.
```

File: data/pages-content/jobs-board.md [hunk 2]
```diff
@@ -392,3 +392,135 @@ Kiwi.com uncovers cheap travel options that others simply can't find.
...
+  - Permanent employment contract - we are counting on you for the long term.
+  - Flexible working hours and the recording of every hour of overtime: You don't have to be on the spot at 7 o'clock in the morning. It is important to us that you show full commitment when you are there.
+  - Flat hierarchies and short decision-making processes - our leaders have open ears and doors.
+  - For the small break in between, there are drinks, organic fruit and snacks for free. For the big appetite, our in-house cafeteria awaits you with vegetarian, vegan and meat dishes made from regional products.
+  - The mood is also good outside our offices, e.g. at the annual company trip, team-building activities, Christmas parties, our monthly games night, and so on. Here we don't just talk about work.
+  - Other benefits: company pension scheme, bicycle leasing and bicycle vouchers or sports allowance.
+
+  You feel challenged and want to participate in shaping the energy transition with us?
+
+  Then apply by e-mail with your application documents: jobs@energymeteo.de
```
Comment (hypha) [inline data/pages-content/jobs-board.md:445]: duplicate to the next line?

File: data/pages-content/jobs-board.md [hunk 3]
```diff
@@ -392,3 +392,135 @@ Kiwi.com uncovers cheap travel options that others simply can't find.
...
+  - Your working day is full of variety, your job is challenging and your team is motivated.
+  - Permanent employment contract - we are counting on you for the long term.
+  - Flexible working hours and the recording of every hour of overtime: You don't have to be on the spot at 7 o'clock in the morning. It is important to us that you show full commitment when you are there.
+  - Flat hierarchies and short decision-making processes - our leaders have open ears and doors.
+  - For the small break in between, there are drinks, organic fruit and snacks for free. For the big appetite, our in-house cafeteria awaits you with vegetarian, vegan and meat dishes made from regional products.
+  - The mood is also good outside our offices, e.g. at the annual company trip, team-building activities, Christmas parties, our monthly games night, and so on. Here we don't just talk about work.
+  - Other benefits: company pension scheme, bicycle leasing and bicycle vouchers or sports allowance.
+
+  You feel challenged and want to participate in shaping the energy transition with us?
+  Then apply by e-mail with your application documents: jobs@energymeteo.de
```
Comment (hypha) [inline data/pages-content/jobs-board.md:523]: duplicate here?

File: data/pages-content/jobs-board.md [hunk 4]
```diff
@@ -392,3 +392,135 @@ Kiwi.com uncovers cheap travel options that others simply can't find.
...
+
+</details>
```
Comment (hypha) [inline data/pages-content/jobs-board.md:525]:
```suggestion
</details>

```

**Review by hypha — COMMENTED** [review]
I think the last bit needs line break after </details>? 
image

File: data/pages-content/jobs-board.md (hunk 2 above)
Comment (Kislovskiy) [reply data/pages-content/jobs-board.md:445]: Screenshot 2023-07-14 at 13 50 15
I think, in shorter view "Apply via email" looks better than "Then apply by e-mail with your application documents", what do you think?

Comment (hypha) [reply data/pages-content/jobs-board.md:445]: absolutely!

**Pull request merged**